docs/HOWTO-run_tests.md

```diff
@@ -14,3 +14,21 @@ GRANT ALL PRIVILEGES ON DATABASE spoke_test TO spoke_test;
 ## SQLite Testing (simpler)
 
 1) Run `npm run test-sqlite`
+
+## End-To-End (Interactive Browser) Testing
+
+1. Remember to set `NODE_ENV=dev` 
+1. Setup DB
+1. Start Spoke Server
```

Comment:
Can we put a sample script here to run, like `npm run dev`

Reply:
Yes, the next PR will have the "final" script which will be `npm run test-e2e`